node/impl/full/gas.go

```diff
@@ -295,31 +296,62 @@ func gasEstimateGasLimit(
 		return -1, xerrors.Errorf("message execution failed: exit %s, reason: %s", res.MsgRct.ExitCode, res.Error)
 	}
 
-	// Special case for PaymentChannel collect, which is deleting actor
-	st, err := smgr.ParentState(ts)
-	if err != nil {
-		_ = err
-		// somewhat ignore it as it can happen and we just want to detect
-		// an existing PaymentChannel actor
-		return res.MsgRct.GasUsed, nil
-	}
-	act, err := st.GetActor(msg.To)
-	if err != nil {
-		_ = err
-		// somewhat ignore it as it can happen and we just want to detect
-		// an existing PaymentChannel actor
-		return res.MsgRct.GasUsed, nil
+	ret := res.MsgRct.GasUsed
+
+	transitionalMulti := 1.0
+	// Overestimate gas around the upgrade
```

Comment:
Can we create an issue to remove (or make more reusable/cleaner/self contained) this after the upgrade epoch?

Reply:
Yes!